tests/app/notifications/rest/test_send_notification.py

```diff
@@ -133,6 +133,65 @@ def test_send_notification_with_placeholders_replaced(notify_api, sample_email_t
             assert response_data['subject'] == 'Jo'
 
 
+@pytest.mark.parametrize('personalisation, expected_body, expected_subject', [
+    (
+        ['Jo', 'John', 'Josephine'],
+        (
+            'Hello \n\n'
+            '* Jo\n'
+            '* John\n'
+            '* Josephine\n'
+            'This is an email from GOV.\u200BUK'
+        ),
+        'Jo, John and Josephine',
+    ),
+    (
+        6,
+        (
+            'Hello 6\n'
+            'This is an email from GOV.\u200BUK'
+        ),
+        '6',
+    ),
+    pytest.mark.xfail((
+        None,
+        ('we consider None equivalent to missing personalisation'),
```

Comment:
i think rather than this xfailing, it should probably assert that the placeholder's still in. maybe split out to a separate test function if that makes sense.

i mean, this is only xfailing because you're setting the value to something we know it won't be - this is currently a "test as documentation" rather than "does our code continue to work in the same way"